js/components/form/base-completer.vue

```diff
@@ -15,22 +15,23 @@ import Vue from 'vue';
 import $ from 'jquery';
 import log from 'logger';
 import BaseError from 'models/error';
+import {isFunction,isString} from 'utils';
```

Comment:
That's the moment I always wonder if importing `utils` and using `utils.isString` after is more pertinent. Relevant thread: https://www.reddit.com/r/javascript/comments/40n0yq/import_of_single_functions_from_modules_good_or/